app/api/groups/get_user_batch_prefixes.go

```diff
@@ -0,0 +1,108 @@
+package groups
+
+import (
+	"net/http"
+
+	"github.com/go-chi/render"
+
+	"github.com/France-ioi/AlgoreaBackend/app/service"
+)
+
+// swagger:model userBatchPrefix
+type userBatchPrefix struct {
+	// required: true
+	GroupPrefix string `json:"group_prefix"`
+	// required: true
+	GroupID int64 `json:"group_id,string"`
+	// Nullable
+	// required: true
+	MaxUsers *int `json:"max_users"`
+	// total number of users in the batches already using this group_prefix
+	// required: true
+	TotalSize int `json:"total_size"`
+}
+
+// swagger:operation GET /groups/{group_id}/user-batch-prefixes groups userBatchPrefixesView
+// ---
+// summary: List user-batch prefixes
+// description: >
+//
+//   Lists the user-batch prefixes  with `allow_new` = 1 matching the input group's ancestors
+//   (i.e., the `group_id` is a descendant of `user_batch_prefixes.group_id`).
```

Comment:
A only managers of the prefix group will be able to use this prefix, only list prefixes whose group has the current user as manager.

Reply:
fixed